support-workers/cloud-formation/src/templates/state-machine.yaml

```diff
@@ -128,20 +128,20 @@ States:
     Type: Choice
     # These choices have a Next set to the steps generated by `supporterPlusProduct` below
     Choices:
-      {{> supporterPlusProductChoice countryCode="UK" currency="GBP" amount=10 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="EU" currency="EUR" amount=10 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="INT" currency="USD" amount=13 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="US" currency="USD" amount=13 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="CA" currency="CAD" amount=13 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="NZ" currency="NZD" amount=17 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="AU" currency="AUD" amount=17 billingPeriod="Monthly" }}
-      {{> supporterPlusProductChoice countryCode="UK" currency="GBP" amount=95 billingPeriod="Annual" }}
-      {{> supporterPlusProductChoice countryCode="EU" currency="EUR" amount=95 billingPeriod="Annual" }}
-      {{> supporterPlusProductChoice countryCode="INT" currency="USD" amount=120 billingPeriod="Annual" }}
-      {{> supporterPlusProductChoice countryCode="US" currency="USD" amount=120 billingPeriod="Annual" }}
-      {{> supporterPlusProductChoice countryCode="CA" currency="CAD" amount=120 billingPeriod="Annual" }}
-      {{> supporterPlusProductChoice countryCode="NZ" currency="NZD" amount=160 billingPeriod="Annual" }}
-      {{> supporterPlusProductChoice countryCode="AU" currency="AUD" amount=160 billingPeriod="Annual" }}
```

Comment:
A good example of where clearing out redundant code would have avoided this as if we removed `currency` as redundant, it would have broken and the bug averted.